### PR TITLE
New setup.py

### PR DESCRIPTION
**Pull request by efficks**
New setup.py script using distutils python library.
Should run on Windows and Linux. Didn't test it on mac.

On Linux add man pages and rename scripts from asciidoc.py to asciidoc
On Windows add a .bat script

I've commit man page files because to generate them on Linux system with a2x you will need some extra executable that you can be possibly missing on your system like xsltproc.
This fixes issue #69

### CODE REVIEW

**Comment by apjanke** [comment]
How does one run the new `setup.py`-based build system? The `autoconf`/`make` files are still there, and the instructions in `INSTALL.txt` still say to use them.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+	if not os.path.exists(scriptdir):
+		os.makedirs(scriptdir)
+	newScripts = []
+	for s in scripts:
+		newpath = os.path.join(scriptdir, os.path.splitext(s)[0])
+		outFh = open(newpath, "w")
+		inFh = open(s, "r")
+		for line in inFh:
+			if line.startswith("CONF_DIR = '"):
+				line = "CONF_DIR = '/usr/local/asciidoc'%s" % os.linesep
```
Comment (apjanke) [inline setup.py:45]: This looks like it's hardcoded the installation location. Is there a way to relocate the installation, as with `./configure --PREFIX=...` using the `make`-based build system?

**Comment by elextr** [comment]
@efficks can confirm, but I think the use-case for this is to enable asciidoc to be put in PYPI.  So its not intended for human consumption as the setup.py is run by Python distutils.  The standard build system should therefore stay as well.

**Comment by efficks** [comment]
I've first add this script to made installation easier on Windows. With this script, it's easier to deploy ASCII doc on any system without installing many other tools like make. It's also multiplatform. With it, it would be easy to deploy on PYPI.
You will also be able to generate setup.exe file on Windows, .deb files to release new versions.
I think it's the way to go for asciidoc and eventually remove makefile.
It also supports 64 bit systems.

**Comment by aerostitch** [comment]
@florentx  It seems this repo will no longer accept PR as the latest release has been cut a while ago but https://github.com/asciidoc/asciidoc-py3 would.
If you still want this PR integrated in asciidoc, do you mind pushing it there please?
Once done I think you can close this one.
Thanks!

**Pull request closed without merging**